modules/network-observability-resource-recommendations.adoc

```diff
@@ -0,0 +1,44 @@
+
+    Restricting or excluding interfaces::  Reduce the overall observed traffic using `spec.agent.ebpf.interfaces` and `spec.agent.ebpf.excludeInterfaces`. By default, the agent fetches all the interfaces in the system, except the ones listed in `excludeInterfaces` and 'lo' (local interface). Note that the interface names might vary according to the CNI used.
+
+The following table documents recommendations for resources depending on workload size. 
+[IMPORTANT]
+====
+These recommended scenarios are tailored to specific workloads and should serve only as a baseline from which adjustments can be made to accommodate your workload needs.
+====
+
+.Resource recommendations
+[options="header"]
+|============================================================================================================================
+|                                     | Extra small (10 Nodes) | Small (25 nodes)  | Medium (65 Nodes) ^[2]^ | Large (120 nodes) ^[2]^
+| *Worker Node Size*                  | m5.xlarge ^[1]^ \| 4 vCPUs\| 16GiB Mem | m5.4xlarge ^[1]^ \| 16 vCPUs\| 64GiB Mem | m5.4xlarge ^[1]^ \| 16 vCPUs\| 64GiB Mem | m5.4xlarge ^[1]^ \| 16 vCPUs\| 64GiB Mem
+| *LokiStack Size*                    | 1x.extra-small         | 1x.small          | 1x.small           | 1x.medium
+| *NetObserv Controller Memory Limit* | 400Mi (default)        | 400Mi (default)   | 400Mi (default)    | 800Mi                
+| *eBPF Sampling Rate*                | 50 (default)           | 50 (default)      | 50 (default)       | 50 (default)
+| *eBPF Memory Limit*                 | 800Mi (default)        | 800Mi (default)   | 2000Mi             | 800Mi (default) 
+| *FLP Mem Limit*                     | 800Mi (default)        | 800Mi (default)   | 800Mi (default)    | 800Mi (default)         
+| *FLP Kafka Partitions*              | N/A                    | 48                | 48                 | 48            
+| *Kafka Consumer Replicas*           | N/A                    | 24                | 24                 | 24
+| *Kafka Brokers*                     | N/A                    | 3 (default)       | 3 (default)        | 3 (default)
+|============================================================================================================================
+1. Tested with AWS m6i images.
+2. In addition to this worker and its controller, 3 infra nodes (size m5.12xlarge) and 1 workload node (size m5.8xlarge) were tested. 
```

Comment:
@nathan-weinberg is this OK?

Reply:
Checked the preview but I don't think it's updated with these changes?

Reply:
No worries @skrthomas - I'm a little confused as to why we are now explicitly stating `m5` machines in the table but have a footnote saying it was tested with `m6`?

Reply:
I think we could just have the table say the CPU and Mem totals, and have the footnote say "Tested with AWS m6i Instances" or something like that - cc @stleerh

Reply:
Ohhh okay...I think the confusion is on my part. I didn't know if I should take away the m5 totally...Looking back at an earlier comment about this section, I think I'm understanding now:
> Should we be so pro-AWS?  What is our response if a customer is using a different provider?  Perhaps it should just specify the vCPU and memory and then put an asterisk there.  Then there could be a footnote that says, we tested with AWS m6i images.

I will remove the image references completely, leave the footnote and the vCPU and memory.

Reply:
@nathan-weinberg Should the footnote about the images also change from m5 -> m6?

Reply:
No worries @skrthomas - yeah the footnote can still be `m6`

Reply:
Cool, I think this is fully addressed now :) but if not, please let me know.

Reply:
`m6i` section

Reply:
Ah, yep. I think your comment and my changes adjusting the footnote crossed in the interwebs. I just pushed that change to make the footnote follow the resources and not preceed in a separate section, which we def don't want...I will adjust the machine names in the footnote for m6i. Thanks for pointing that out. I just simply replaced 'm5' with 'm6'